src/app.js

```diff
@@ -108,101 +108,15 @@ io.on('connection', (socket) => {
 		if (game != undefined) {
 			if (game.findPlayer(socket.id).blindValue == 'Big Blind' && game.roundData.bets.length == 1) game.bigBlindWent = true;
 			if (data.move == 'fold') {
-				let preFoldBetAmount = 0;
-
-				let roundDataStage = game.roundData.bets[game.roundData.bets.length - 1].find(a => a.player == game.findPlayer(socket.id).username);
-				if (roundDataStage != undefined && roundDataStage.bet != 'Fold') {
-					preFoldBetAmount += roundDataStage.bet;
-				}
-				game.findPlayer(socket.id).setStatus('Fold');
-				game.foldPot = game.foldPot + preFoldBetAmount;
-				if (game.roundData.bets[game.roundData.bets.length - 1].some(a => a.player == game.findPlayer(socket.id).username)) {
-					game.roundData.bets[game.roundData.bets.length - 1] = game.roundData.bets[game.roundData.bets.length - 1].map(a => a.player == game.findPlayer(socket.id).username ? { player: game.findPlayer(socket.id).getUsername(), bet: 'Fold' } : a);
-				} else {
-					game.roundData.bets[game.roundData.bets.length - 1].push({ player: game.findPlayer(socket.id).getUsername(), bet: 'Fold' });
-				}
-				game.lastMoveParsed = { 'move': 'Fold', 'player': game.findPlayer(socket.id) };
-				game.moveOntoNextPlayer();
+				game.fold();
```

Comment:
the `fold` function should also be passed a socket, since it needs to change the player's status